apps/coscrad-frontend/src/components/resources/digital-text/new-page-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { isNonEmptyString } from '@coscrad/validation-constraints';
+import { Button, Stack, TextField } from '@mui/material';
+import { useState } from 'react';
+
+interface NewPageFormProps {
+    onSubmitPageIdentifier: (pageIdentifier: string) => void;
+    existingPageIdentifiers: string[];
+}
+
+export const NewPageForm = ({
+    onSubmitPageIdentifier,
+    existingPageIdentifiers,
+}: NewPageFormProps) => {
+    const [pageIdentifier, setPageIdentifier] = useState<string>('');
+
+    // remove leading and trailing white space
+    const parsedPageIdentifier = pageIdentifier.trim();
```

Comment:
This is not quite right. We want to make sure there are no white spaces, tabs, or new lines anywhere in the page identifier.

Reply:
Fixed.